src/explore-education-statistics-common/src/services/publicationService.ts

```diff
@@ -10,6 +10,23 @@ import { contentApi } from './api';
 
 export type ReleaseApprovalStatus = 'Draft' | 'HigherLevelReview' | 'Approved';
 
+// TODO EES-3517 a guess at the new type for the find stats page.
+export interface PublicationSummaryWithRelease {
```

Comment:
Would move this next to `PublicationSummary` as they're similar in name/context.